src/marine/emc_ice2ioda.py

```diff
@@ -28,7 +28,9 @@
 class Observation(object):
 
     def __init__(self, filename, thin, date, writer):
-        print(date)
+        print('date = ', date)
+        global date2
```

Comment:
@YanHao-NOAA , why not just add date2 as data member of observation instead of making it a global variable?

Reply:
Hi, guillaume, thank you for pointing this out. I changed the code to pass date information to read function, instead of using  the global variable. Please check my updated code. Thanks!